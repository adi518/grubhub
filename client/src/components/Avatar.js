import React from 'react';
import styled from 'styled-components';

const StyledAvatar = styled.div`
  margin-right: 1rem;

  img {
    border-radius: 50%;
  }
`;

const Avatar = ({ user }) => (
  <StyledAvatar>
    <img src={user.profile_image_url} alt={user.name} />
  </StyledAvatar>
);

export default Avatar;
