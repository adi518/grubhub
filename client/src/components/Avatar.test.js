import React from 'react';
import Avatar from './Avatar';
import { render } from '@testing-library/react';

test('Avatar component', () => {
  const { asFragment } = render(
    <Avatar
      user={{
        name: 'Driver Man',
        profile_image_url:
          'http://pbs.twimg.com/profile_images/1201972977396379648/5ZQCnQOa_normal.jpg'
      }}
    />
  );
  expect(asFragment()).toMatchSnapshot();
});
