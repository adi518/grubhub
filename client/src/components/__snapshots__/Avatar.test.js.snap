// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar component 1`] = `
<DocumentFragment>
  <div
    class="sc-bdVaJa gHrgfm"
  >
    <img
      alt="Driver Man"
      src="http://pbs.twimg.com/profile_images/1201972977396379648/5ZQCnQOa_normal.jpg"
    />
  </div>
</DocumentFragment>
`;
